### PR TITLE
feat(coc): add COC script resource

### PR DESCRIPTION
**Pull request by ShiChangkuo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

* Add new resource **huaweicloud_coc_script**which is only internal used.

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ TF_ACC=1 go test -v ./huaweicloud/services/acceptance/coc -v -run TestAccScript_basic
=== RUN   TestAccScript_basic
=== PAUSE TestAccScript_basic
=== CONT  TestAccScript_basic
--- PASS: TestAccScript_basic (37.52s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/coc       40.855s
```

### CODE REVIEW

File: docs/incubating/coc_script.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+* `name` - (Required, String, ForceNew) Specifies the name of the script. The value can contains 3 to 64 characters,
+  including letters, digits, hyphens (-), and underscores (_). Changing this creates a new resource.
+
+* `description` - (Required, String) Specifies the description of the script.
+  The value can consist of up to 256 characters.
+
+* `risk_level` - (Required, String) Specifies the risk level. The valid values are **LOW**, **MEDIUM** and **HIGH**.
+
+* `version` - (Required, String) Specifies the version of the script. For example, *1.0.0* or *1.1.0*.
```
Comment (Lance52259) [inline docs/incubating/coc_script.md]:
```suggestion
* `version` - (Required, String) Specifies the version of the script. For example, **1.0.0** or **1.1.0**.
```

File: huaweicloud/services/acceptance/coc/resource_huaweicloud_coc_script_test.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  parameters {
+    name        = "name"
+    value       = "world"
+    description = "the first parameter"
+  }
+  parameters {
+    name        = "company"
+    value       = "Huawei"
+    description = "the second parameter"
+	sensitive   = true
```
Comment (Lance52259) [inline huaweicloud/services/acceptance/coc/resource_huaweicloud_coc_script_test.go]:
```suggestion
    sensitive   = true
```

File: docs/incubating/coc_script.md (hunk 1 above)
Comment (ShiChangkuo) [reply docs/incubating/coc_script.md]: fixed

File: huaweicloud/services/acceptance/coc/resource_huaweicloud_coc_script_test.go (hunk 2 above)
Comment (ShiChangkuo) [reply huaweicloud/services/acceptance/coc/resource_huaweicloud_coc_script_test.go]: fixed

**Pull request merged**